Tests/Exporter/ExporterTest.php

```diff
@@ -12,21 +12,30 @@
 namespace Sonata\CoreBundle\Tests\Exporter;
 
 use Exporter\Source\ArraySourceIterator;
+use Exporter\Writer\CsvWriter;
+use Exporter\Writer\JsonWriter;
+use Exporter\Writer\XlsWriter;
+use Exporter\Writer\XmlWriter;
 use Sonata\CoreBundle\Exporter\Exporter;
 
 class ExporterTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * @expectedException RuntimeException
-     */
     public function testFilter()
     {
+        $this->setExpectedException('RuntimeException', 'Invalid "foo" format');
         $source = $this->getMock('Exporter\Source\SourceIteratorInterface');
+        $writer = $this->getMock('Exporter\Writer\WriterInterface');
 
-        $exporter = new Exporter();
```

Comment:
we should keep a test with an empty constructor, tot test if the fallback works, shouldn't we?